ios/A0Auth0.m

```diff
@@ -37,58 +19,43 @@ - (dispatch_queue_t)methodQueue
 }
 
-RCT_EXPORT_METHOD(getCredentials:(NSString *)scope minTTL:(NSInteger)minTTL parameters:(NSDictionary *)parameters resolver:(RCTPromiseResolveBlock)resolve rejecter:(RCTPromiseRejectBlock)reject) {
-    [self.credentialsManagerBridge getCredentialsWithScope:scope minTTL:minTTL parameters:parameters resolve:resolve reject:reject];
+RCT_EXPORT_METHOD(getCredentials:(NSString *)scope minTTL:(NSInteger)minTTL parameters:(NSDictionary *)parameters forceRefresh:(BOOL)forceRefresh resolver:(RCTPromiseResolveBlock)resolve rejecter:(RCTPromiseRejectBlock)reject) {
+    [self.nativeBridge getCredentialsWithScope:scope minTTL:minTTL parameters:parameters forceRefresh:forceRefresh resolve:resolve reject:reject];
 }
 
 RCT_EXPORT_METHOD(hasValidCredentials:(NSInteger)minTTL resolver:(RCTPromiseResolveBlock)resolve rejecter:(RCTPromiseRejectBlock)reject) {
-    [self.credentialsManagerBridge hasValidCredentialsWithMinTTL:minTTL resolve:resolve];
+    [self.nativeBridge hasValidCredentialsWithMinTTL:minTTL resolve:resolve];
 }
 
 RCT_EXPORT_METHOD(clearCredentials:(RCTPromiseResolveBlock)resolve rejecter:(RCTPromiseRejectBlock)reject) {
-    [self.credentialsManagerBridge clearCredentialsWithResolve:resolve reject:reject];
+    [self.nativeBridge clearCredentialsWithResolve:resolve reject:reject];
 }
 
 RCT_EXPORT_METHOD(enableLocalAuthentication:(NSString *)title cancelTitle:(NSString *)cancelTitle fallbackTitle:(NSString *)fallbackTitle evaluationPolicy:(NSInteger)evaluationPolicy) {
-    [self.credentialsManagerBridge enableLocalAuthenticationWithTitle:title cancelTitle:cancelTitle fallbackTitle:fallbackTitle evaluationPolicy: evaluationPolicy];
+    [self.nativeBridge enableLocalAuthenticationWithTitle:title cancelTitle:cancelTitle fallbackTitle:fallbackTitle evaluationPolicy: evaluationPolicy];
 }
 
-RCT_EXPORT_METHOD(showUrl:(NSString *)urlString
-                  usingEphemeralSession:(BOOL)ephemeralSession
-                  closeOnLoad:(BOOL)closeOnLoad
-                  callback:(RCTResponseSenderBlock)callback) {
-    if (@available(iOS 11.0, *)) {
-        self.sessionCallback = callback;
-        self.closeOnLoad = closeOnLoad;
-        [self presentAuthenticationSession:[NSURL URLWithString:urlString] usingEphemeralSession:ephemeralSession];
-    } else {
-        [self presentSafariWithURL:[NSURL URLWithString:urlString]];
-        self.sessionCallback = callback;
-        self.closeOnLoad = closeOnLoad;
-    }
+RCT_EXPORT_METHOD(webAuth:(NSString *)scheme state:(NSString *)state nonce:(NSString *)nonce audience:(NSString *)audience scope:(NSString *)scope connection:(NSString *)connection maxAge:(NSInteger)maxAge organization:(NSString *)organization invitationUrl:(NSString *)invitationUrl  leeway:(NSInteger)leeway ephemeralSession:(BOOL)ephemeralSession additionalParameters:(NSDictionary *)additionalParameters resolver:(RCTPromiseResolveBlock)resolve rejecter:(RCTPromiseRejectBlock)reject) {
```

Comment:
Is `scheme` needed in the method signature for the iOS platform?

Reply:
The method signature for iOS and Android has to be same. So we are adding the missing parameters as well.